backend/server.py

```diff
@@ -43,7 +45,17 @@ async def upload(file: UploadFile = Form(...),
 
         # Delete the temporary chunks directory
         shutil.rmtree(CHUNKS_DIR.parent)
+        
+        # STT Process
+        stt_file = get_stt_from_path(path=final_file_path, is_escaped=False)
+        # Create the stt_files directory if it doesn't exist
+        STT_DIR.mkdir(parents=True, exist_ok=True)
+        stt_file_path = STT_DIR / filename
+        with open(f'{stt_file_path}.txt', 'w') as file:
+            file.write(stt_file)
+            print('Finished saving stt file')
```

Comment:
convert this to log using [python's logger](https://docs.python.org/3/howto/logging.html)